### DISCUSSION TITLE
Does bedrock autologin work?

### DISCUSSION THREAD

**Discussion by paj1s** [games647/FastLogin#748] [Q&A] [answered]
I tried to setup the plugin many times and im sure i have set it up correctly but bedrock players still wont get autologed. Does this function work or is the problem still somewhere in my config?

**Comment by TuxCoding** [comment]
Please explain your setup like server software, configuration etc.

**Comment by paj1s** [comment, marked as answer]
Waterfall + paper servers. Plugins that matters: geyser on proxy, floodgate on proxy + servers, fastlogin on proxy + lobby server, authme on lobby server. All software is up to date. I tried almost all variants of config which should autologin the bedrock player including all on and also all on and floodgate related to no-conflict.

**Reply by TuxCoding** [reply]
Please post the log that includes the startup and the event when a player logs in.

**Reply by paj1s** [reply]
Oh sorry, i fixed it about 10 hours ago and forgot to mention, i have downloaded the latest versions, removed old configs and set new with care. Still autologin wasn't working so i flushed authme database and now it's working like a charm!
